buildpipeline/pipeline.json

```diff
@@ -130,6 +130,29 @@
           }
         },
         {
+          "Name": "Core-Setup-CrossBuild",
+          "Parameters": {
+            "PB_Architecture": "arm"
```

Comment:
Just to be explicit, you should pass in values for PB_DockerTag & TargetRid here, like below (not strictly necessary, but makes readability easier).